ruby/spec/spec_helper.rb

```diff
@@ -1,6 +1,9 @@
 require 'bundler/setup'
 require 'do_username'
 
+require 'simplecov'
+SimpleCov.start
```

Comment:
```suggestion
require 'simplecov'
SimpleCov.minimum_coverage 100
SimpleCov.start

require 'do_username'
```

Simplecov must be included before application code, so that it can track LOC correctly. We can also require 100% coverage by default